scripts/website.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+mkdir -p logs
+
+# This downloads a set of key-value env variables, but blocks any requests that aren't from GitHub Actions
+curl -g https://us-central1-clarity-design-system.cloudfunctions.net/actions -o .env
+# dotenv loads up the env variables into the shell, then deploys through Netlify
+
+# Deploy a preview that can be promoted to production when we are ready
+node -r dotenv/config -- ./node_modules/.bin/netlify deploy --json --dir=./dist/website --message="Angualr Branch - Website - $GITHUB_REF@$GITHUB_SHA" --site "clarity.design" > ./logs/clarity.design.netlify.json
+
+echo "Clarity Design Netlify"
```

Comment:
You don't need the two lines below .. cause we don't build `clarity.design` here so on `Github CI` run there will be no one to create it.

Reply:
This will actually need to be updated when we deploy the website on the subdomain, but that isn't setup yet.